src/Questionnaire/Questionnaire.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react'
+import { questionnaires } from '../data/questionnaires'
+
+export default function Questionnaire() {
+  const currentQuestionnaire = questionnaires[0]
+  const { questions } = currentQuestionnaire
+  const [currentQuestionIndex, setcurrentQuestionIndex] = useState(0)
+  const [answers, setAnswers] = useState([])
+  const [result, setResult] = useState(0)
+
+  useEffect(() => {
+    setResult(countYes(answers))
+  }, [answers])
+
+  return currentQuestionIndex < questions.length ? (
+    <>
+      <h3>Frage {currentQuestionIndex + 1} / 8 </h3>
+      <h2>{questions[currentQuestionIndex]}</h2>
+      <button onClick={handleAnswer} data-js="no">
+        (Eher) Nein
+      </button>
+      <button onClick={handleAnswer} data-js="yes">
```

Comment:
Potentially you could add the yes/no as an argument to the handleAnswer function in order to omit the data-js attribute
```suggestion
      <button onClick={() => handleAnswer('yes')}>
```

Reply:
Good idea. I think the arrow function is necessary, otherwise the function would be called directly after rendering the component, not just on click. I changed the code like you suggested.